camera/camera_balsaq.hh

```diff
@@ -28,6 +28,7 @@ class CameraBq : public BalsaQ {
   void init(const Config& config);
   void loop();
   void shutdown();
+  const static uint loop_delay_microseconds = 50000;
```

Comment:
Does this work? This appears to shadow the base class’s `loop_delay_microseconds`

Reply:
You should be able to just assign a value to the base class’s `loop_delay_microseconds` Without redefining the variable.

Reply:
Done.